src/routes/Board/Board.tsx

```diff
@@ -40,8 +42,12 @@ function Board() {
             <>    
                 {waitingUser}
                 <BoardComponent name={state.board.data!.name} boardstatus={boardstatus}>
-                        {state.board.data!.columns.map((column:any) => (
-                            <Column color="pink" key={column.id} columnId={column.id}/>
+                        {state.board.data!.columns.map((column) => (
+                            <Column key={column.id} id={column.id!} name={column.name} color={column.color}>
+                                {state.notes.filter((note) => note.columnId === column.id).map((note) =>
```

Comment:
You could use the NoteClientModel type

Reply:
The `state.notes` already have the correct type, so there's no need to write it more explicit?